src/components/ha-push-notifications-toggle.js

```diff
@@ -69,6 +70,8 @@ class HaPushNotificationsToggle extends EventsMixin(PolymerElement) {
     // it to get into a loop and crash the page.
     if (!pushSupported) return;
 
+    this.pushChecked = event.target.checked;
```

Comment:
Is this needed? It should be a 2-way binding.

Reply:
Yes, that's why I added it ;)

Reply:
That should be fixed in `ha-switch` then probably...

Reply:
Can do debugging tomorrow

Reply:
Another way to fix it is to rewrite this element to `lit` 😆

Reply:
I actually thought about that but figured it was overkill for this simple change.
But sure... Why not.